extensions/wikia/AdMixExperiment/scripts/noVideo1.js

```diff
@@ -0,0 +1,269 @@
+require(['wikia.throttle'], function (throttle) {
+	$(function () {
+
+		var $adAndRecircWrapper;
+		var $topRightAd = $('#TOP_RIGHT_BOXAD');
+		var $topRightAdWrapper = $('#top-right-boxad-wrapper');
+		var $footer = $('#WikiaFooter');
+		var $rail = $('#WikiaRail');
+		var $recirc;
+		var $recircWrapper;
+		var $visibleElementBeforeWrapper;
+		var $window = $(window);
+
+		var adMixRecircWrapperHeight;
+		var bottomMargin = 20;
+		var breakpointSmall = 1023;
+		var globalNavigationHeight = $('#globalNavigation').outerHeight(true);
+		var gapSize = 200;
+		var recircEnabled = true;
+		var topRightAdHeight = 250;
+		var noRecircAdBottomPosition = 5;
+		var viewportHeight = $(window).height();
+		var viewportWidth = $(window).width();
+		var visibleElementBeforeWrapperHeight;
+
+		// TODO that should be set to true for ad mix variant XW-3156
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)